src/components/AboutApp.js

```diff
@@ -0,0 +1,34 @@
+import { h } from 'preact'
+import { useI18n, useSoftkey } from 'hooks'
+
+export const AboutApp = () => {
+  const i18n = useI18n()
+  const APP_VERSION = JSON.stringify(require('../../package.json').version).slice(1, -1)
```

Comment:
I tried using the `DefinePlugin` global variable here (mirroring [this use of it](https://github.com/wikimedia/wikipedia-kaios/blob/master/src/hooks/useTracking.js#L107)), but it wasn't working for some reason.

Note I'm using a relative path within `require` because of the same issue [brought up here](https://github.com/wikimedia/wikipedia-kaios/pull/162#discussion_r392731983)

Reply:
you can use 

```html
{/* eslint-disable-next-line no-undef */}
<p>{APP_VERSION}</p>
```

within the jsx line

Reply:
Right, you need to use the already-defined constant because package.json is not going to be available when the app runs on the phone.

Reply:
Perfect